src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -980,6 +997,7 @@ export default {
 				&& this.isActivelyLoggedIn
 				&& this.teamId
 				&& !this.teamJoinStatus
+				// I think this is the general area to show the checkout after login for all users
```

Comment:
All of these checks "should" run any time the page is loaded... I'm guessing that we aren't firing `handleBasketValidation` for the matching page case...